src/applications/disability-benefits/all-claims/config/form.js

```diff
@@ -203,32 +218,37 @@ const formConfig = {
         disabilitiesOrientation: {
           title: '',
           path: 'disabilities/orientation',
+          // Only show the page if both (or potentially neither) options are chosen on the claim-type page
+          depends: formData =>
+            newAndIncrease(formData) || noClaimTypeSelected(formData),
           uiSchema: { 'ui:description': disabilitiesOrientation },
           schema: { type: 'object', properties: {} },
         },
         ratedDisabilities: {
           title: 'Existing Conditions (Rated Disabilities)',
           path: 'disabilities/rated-disabilities',
-          depends: hasRatedDisabilities,
+          depends: formData =>
+            hasRatedDisabilities(formData) && !newOnly(formData),
```

Comment:
🤔 maybe this is just me but I had to think about `newOnly` a bit. What about `newConditionsOnly` or something to more clearly indicate what's new? 🤷‍♂️

Reply:
Ah, there you go. I like `newConditionsOnly`. :+1:
I chose `newOnly` instead of `isClaimingOnlyNewConditions` because...well, y'know...Typing.